index.js

```diff
@@ -1,58 +1,158 @@
 const svelte = require('svelte/compiler');
-const deasync = require('deasync');
+const { execSync } = require('child_process');
+const { transformSync, transform } = require('@babel/core');
 
 const styleRegex = /<style[^>]*>[\S\s]*?<\/style>/g;
+const dynImportRegex = /import\(/g;
```

Comment:
```suggestion
const dynImportRegex = /\bimport\(/g;
```
Maybe with word boundary to *guarantee* that you only match the reserved `import` keyword in JS?